ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -46,7 +46,12 @@ object Main {
       case LogEncoder.Plain => () // This is the default
       case LogEncoder.Json =>
         Logging.setUseJsonLogEncoderSystemProp()
-        Logging.reconfigure(getClass, "logback.xml")
+        val path: PathKind =
```

Comment:
The fact that you only do this in the JSON API seems a bit worrying. We handle logback files the same for all components. Should we move the logic into `reconfigure`?

Reply:
That was my thought too, but so far I see only the json api does this because no other service spits out json logs yet if I remember correctly.

Considering that the system prop is named same for every other service I think this should be safe to move there :thinking:

Reply:
I’m hoping that the fact that no other sevrice supports this is a temporary state and eventually everything produces json logs :)

Reply:
ok adressed with e795cfc172fdc5fc1ce338a6cc9ad992a6f7010c

fine for you @cocreature ?